schemas/cgTranscripts.js

```diff
@@ -0,0 +1,15 @@
+var mongoose = require("mongoose");
+var Schema = mongoose.Schema;
+
+var cgTranscriptSchema = new Schema({
+  bitsId: String,
+  date: Date,
+  applicationType: String,
+  status: String,
+  email: String,
```

Comment:
Either use the `email` or `idNumber` to identify applications. Using both of them is redundant.

Reply:
Email is a used as a store of data, not as an indentifier.